### ISSUE TITLE
Compatibility issue Universal Graves?

### ISSUE THREAD

**Issue by Zarepheth** [LordDeatHunter/FabricWaystones#57]
When I die and one or more Universal Graves exist, holding my items. I can no longer use the way stones. Clicking a way stone will open the interface. But upon selecting a destination, the UI closes, leaving me where I was. Other way stone operations, like breaking, placing, renaming a way stone all work. And I can add new way stones to my network. I just cannot travel via way stone until all my graves are broken. This makes it very difficult to return to my grave!

**Comment by LordDeatHunter** [comment]
1. Do you have the requirements for the teleportation (items, xp, levels or whatever you have it set to)
2. Does anything appear in the console/log file when the waystone closes?

**Comment by Zarepheth** [comment]
I didn't think there were any requirements, but upon checking the config files, that might be my problem:

This is my `waystones/config.json` file:
```jsonc
{
  "generate_in_villages": true,
  "consume_infinite_knowledge_scroll_on_use": false,
  "can_owners_redeem_payments": false,
  "cost_amount": 1,
  "cost_type": "level",
  "cost_item": "minecraft:ender_pearl",
  "waystone_block_hardness": 4,
  "waystone_block_required_mining_level": 1.0,
  "recipes": {  /* cut for brevity */  }
}
```

So far, all my way stones are in the Overworld - most due to spawning in villages. One from crafting. And some due to a duplication bug when an update caused all the way stones to be waterlogged (and I broke them with a Tech Reborn basic drill, which only broke half the way stone (but each half gave me a whole way stone when I picked them up). I've not tried breaking a water logged way stone since water logged fix.
